### PR TITLE
Release 84

### PR DESCRIPTION
**Pull request by lawrence-dxw**
- (chore) Copy filesystem FDL into the database
- (fix) Disable in2csv's type inference, as it can mis-infer
- (fix) Align development Python version with that of production
- Validate presence of salesforce_id on suppliers
- (chore) Amend the docs for getting production DB backups
- (fix) Don't use gzip when importing database backups
- (chore) Update the DB restoration docs to work on Docker
- (fix) Make db:restore work under Docker
- (chore) Update documentation for grabbing a database dump
- (chore) Fresh docker builds will provision node_module dependencies
- (chore) Only build the docker image for the codebase once
- (fix) Docker setup script treated as a shell file
- (chore) Add a pull request template for GitHub
- (chore) Update the default ENV variables

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [release-84] - 2019-08-19
```
Comment (lawrence-dxw) [inline CHANGELOG.md]: Anyone understand why this is not causing an automatic hyperlink like the other headings in this file are?

Comment (jcoglan) [reply CHANGELOG.md]: I think you need to add the release to the list of links at the bottom of the document.

Comment (lawrence-dxw) [reply CHANGELOG.md]: Ah, thank you!

Comment (lawrence-dxw) [reply CHANGELOG.md]: (fixed it)

**Pull request merged**